src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -125,7 +121,7 @@ public ExtensionsRunner() throws IOException {
             .put(TransportSettings.BIND_HOST.getKey(), extensionSettings.getHostAddress())
             .put(TransportSettings.PORT.getKey(), extensionSettings.getHostPort())
             .build();
-        this.customSettings = Collections.emptyList();
+        ExtensionsRunner.customSettings = Collections.emptyList();
```

Comment:
It is almost always an error to set a static variable in a constructor.  If you instantiate this class multiple times they will all try to set the same value.

Reply:
Same answer as first comment, I will delete the static in my next commit.